crates/pink/src/runtime/extension.rs

```diff
@@ -66,6 +67,8 @@ pub fn get_side_effects() -> ExecSideEffects {
                                 error!("Contract emitted an invalid pink event");
                             }
                         }
+                    } else {
+                        result.ink_events.push((address, data));
```

Comment:
Should we push link events even if `event.topics.len() != 1`?

Reply:
Should